ETC.tsv

```diff
@@ -11012,7 +11012,7 @@ ETC_20150414_011013	Recipe - Katzbalger
 ETC_20150414_011014	Recipe - Dress Sword
 ETC_20150414_011015	{memo X}Recipe - Crude Cane Rover
 ETC_20150414_011016	Recipe - Aras' Long Rod
-ETC_20150414_011017	Recipe - Snake Rod
+ETC_20150414_011017	Recipe - Serpentine Rod
```

Comment:
If you're changing the original from Snake to Serpentine, you have to let IMC know, because the wiki still says Snake.